cmd/prepare_worker.go

```diff
@@ -186,6 +187,22 @@ func (b *workerBuilder) buildTypeScript(ctx context.Context, baseDir string) (sd
 			return nil, fmt.Errorf("version not found in package.json")
 		}
 	}
+
+	// Prep the TypeScript runner to be built
+	cmd := exec.Command("npm", "install")
+	cmd.Dir = "./workers/typescript"
```

Comment:
This and the subsequent one should use abspath after joining with `baseDir` like the other commands do to make sure this isn't dependent on exactly where you run the command from

Reply:
Oops, you're right. I threw this together quickly just to get it working/unblock myself and then did not revisit. Thanks for flagging it.